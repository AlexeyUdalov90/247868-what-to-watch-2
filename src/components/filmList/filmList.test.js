import React from 'react';
import renderer from 'react-test-renderer';

import FilmList from './filmList.jsx';

it(`FilmList correctly renders after relaunch`, () => {
  const tree = renderer.create(<FilmList films={[]} activeItem={0} onChangeActiveItem={jest.fn()} onClickTitle={jest.fn()} />).toJSON();

  expect(tree).toMatchSnapshot();
});
