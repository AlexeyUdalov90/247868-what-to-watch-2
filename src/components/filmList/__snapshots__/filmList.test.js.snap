// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmList correctly renders after relaunch 1`] = `null`;
